src/cpp/cpp_typecheck_compound_type.cpp

```diff
@@ -888,9 +888,9 @@ void cpp_typecheckt::typecheck_friend_declaration(
     // typecheck ftype
 
     // TODO
-//    typecheck_type(ftype);
-//    assert(ftype.id()==ID_symbol);
-//    symbol.type.add("#friends").move_to_sub(ftype);
+    //    typecheck_type(ftype);
+    //    assert(ftype.id()==ID_symbol);
+    //    symbol.type.add("ID_C_friends").move_to_sub(ftype);
```

Comment:
I'm not sure if updating commented out TODO code is better or worse than not updating it.  Hmmm... I think updating is probably the right thing to do.